ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -6,8 +6,10 @@ classes:
 
     PhysicalVolume:
         label: Physical Volume
-        impacts:
-            - volumeGroup
+        dynamicview_views: [service_view]
+        dynamicview_relations:
+            impacts:
+                - volumeGroup
```

Comment:
The HardDisk impact adapter creates a relationship to its physical volumes. The reverse of that relationship needs to be defined here.